daemon/README.md

```diff
@@ -346,6 +346,10 @@ List of available options:
 * `RGW_CIVETWEB_PORT` is the port to which civetweb is listening on (DEFAULT: 80)
 * `RGW_NAME`: default to hostname
 
+Administration via [radosgw-admin](http://docs.ceph.com/docs/infernalis/man/8/radosgw-admin/) from the Docker host:
+
+`docker exec <containerId> radosgw-admin -n client.rgw.$(hostname) -k /var/lib/ceph/radosgw/$(hostname)/keyring <commands>`
```

Comment:
Can you just clarify that `client.rgw.$(hostname)` is default but it might change depending on the value you give to `RGW_NAME`? Thanks!